bin/powershell/metric-windows-disk.ps1

```diff
@@ -41,51 +38,24 @@ if ($UseFullyQualifiedHostname -eq $false) {
     $Path = [System.Net.Dns]::GetHostEntry([string]"localhost").HostName.toLower()
 }
 
-$perfCategoryID = Get-PerformanceCounterByID -Name 'PhysicalDisk'
-$localizedCategoryName = Get-PerformanceCounterLocalName -ID $perfCategoryID
-
-[void]$counters.Add('Avg. Disk Bytes/Read')
-[void]$counters.Add('Avg. Disk Bytes/Write')
-[void]$counters.Add('Avg. Disk sec/Read')
-[void]$counters.Add('Avg. Disk sec/Write')
-[void]$counters.Add('Current Disk Queue Length')
-
-foreach ($ObjDisk in (Get-Counter -Counter "\$localizedCategoryName(*)\*").CounterSamples) {
-
-   if ($instances.ContainsKey($ObjDisk.InstanceName) -eq $false) {
-        
-        if ($ObjDisk.InstanceName.ToLower() -ne '_total') {
-            $disk = $ObjDisk.InstanceName
-            $disk = $disk.Remove(0,1)
-            $disk = $disk.Replace(":","")
-            $disk = $disk.Trim()
-            $instances.Add($ObjDisk.InstanceName,$disk.toUpper())
-        }
-        
-   }
-    
-}
-
-foreach ($diskkey in $instances.Keys) {
-
-    $diskname = $instances.$diskkey
-
-    foreach ($counter in $counters) {
+$driveArray=(Get-CimInstance -ClassName Win32_PerfFormattedData_PerfDisk_PhysicalDisk | Where Name -NotLike '_Total' |Select -ExpandProperty Name)
```

Comment:
previously this accounted for localizing the counter name I dont think this will work if windows is not set to english.